apps/src/templates/instructions/ThreeColumns.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+/**
+ * A component that lays out children in three columns (left, center, right),
+ * where left/right widths are specified and center is the remainder. This is
+ * done largely to separate some of the semi-complicated styling.
+ */
```

Comment:
Perhaps a little bit weird to have a separate component for this, but it felt like a good way to have all the styling for it in one place where it could mostly be ignored.